src/OpenSEE/Scripts/TSX/jQueryUI Widgets/Common.tsx

```diff
@@ -93,7 +93,7 @@ export const WidgetWindow: React.FunctionComponent<IwindowProps> = (props) => {
                 props.setPosition(top, left);
             };
         }
-    });
+    }, [refWindow.current]);
```

Comment:
Didn't this whole file get removed? I don't think we are suing this anywhere anymore - the widgets should all be in Drawers